src/plugins/intel_cpu/src/nodes/deconv.cpp

```diff
@@ -521,6 +521,10 @@ void Deconvolution::getSupportedDescriptors() {
             createDescriptor({in_candidate}, {out_candidate});
         }
     }
+    OPENVINO_ASSERT(!descs.empty(),
```

Comment:
Do we really need this check?
What happens when it is missed?

Reply:
yes, I think so, it is moved from inner loop.

Reply:
I mean in other oneDNN type of nodes we never check if descs is empty in the end of getSupportedDescriptors.
What is the difference here?

Reply:
There is no difference, this checking is used to make sure the descs are not empty.
It's fine to remove this checking.